app/src/main/java/org/mozilla/firefox/vpn/apptunneling/AppTunnelingUtils.kt

```diff
@@ -7,8 +7,10 @@ fun ApplicationInfo.isSystemApp(): Boolean {
     // For the apps on our devices, FLAG_SYSTEM is always set if FLAG_UPDATED_SYSTEM_APP is set.
     // Being uncertain if it's possible an app can have only FLAG_UPDATED_SYSTEM_APP being set, we
     // defensively check both bits here
-    val systemFlag = ApplicationInfo.FLAG_SYSTEM or ApplicationInfo.FLAG_UPDATED_SYSTEM_APP
```

Comment:
I think we should make two separate methods: isSystemApp() and isUpdatedSystemApp(), and make the usage in AppTunnelingRepositoy become

`filter { includeInternalApps || !is.isSystemApp() || isUpdatedSystemApp() }`

since whether an app is a system app is a fact instead of something we can give our own definition to. Also, the flag is named "FLAG_UPDATED_SYSTEM_APP", which means it's still counted as a system app, which will make current isSystemApp() implementation more confusing.